README.md

```diff
@@ -60,3 +60,4 @@ Code is minified using: http://marijnhaverbeke.nl/uglifyjs
 * v 1.3.x - Debounce resizing events, more accurate space width estimate
 * v 1.4.0 - Add support for nested tags (rileyjshaw)
 * v 1.5.0 - Re-balance text on resize for manually triggered selectors (rileyjshaw)
+* v 1.6.0 - Add balanceTextUpdate() method (rileyjshaw)
```

Comment:
Can you also add a paragraph + code sample in README.md explaining usage of new API call?